config/secrets.yml

```diff
@@ -50,7 +50,7 @@ common: &common
   vapid_private_key: <%= ENV['VAPID_PRIVATE_KEY'] %>
   recaptcha_v3_secret_key: <%= ENV['RECAPTCHA_V3_SECRET_KEY'] %>
   recaptcha_v2_secret_key: <%= ENV['RECAPTCHA_V2_SECRET_KEY'] %>
-  login_token_time_limit: <%= ENV.fetch('LOGIN_TOKEN_TIME_LIMIT', '30').to_i.abs.minutes %>
+  login_token_time_limit: <%= ENV.fetch('LOGIN_TOKEN_TIME_LIMIT', '72').to_i.abs.minutes %>
```

Comment:
@bodhish I guess the plan was to make this 72 hours?

Reply:
Done 😺